'dubbo-serialization-gson' maven package error

- [x] I have searched the [issues](https://github.com/apache/incubator-dubbo/issues) of this repository and believe that this is not a duplicate.
- [x] I have checked the [FAQ](https://github.com/apache/incubator-dubbo/blob/master/FAQ.md) of this repository and believe that this is not a duplicate.

### Environment

* Dubbo version: 2.7.2-SNAPSHOT
* Operating System version: mac os
* Java version: 1.8.0_131
* maven version: Apache Maven 3.3.9 

### Steps to reproduce this issue

maven package -Dmaven.test.skip

`[ERROR]     Non-resolvable parent POM for org.apache.dubbo:dubbo-serialization-gson:[unknown-version]: Could not find artifact org.apache.dubbo:dubbo-serialization:pom:2.7.2-SNAPSHOT and 'parent.relativePath' points at wrong local POM @ line 19, column 13 -> [Help 2]`

--- a/dubbo-serialization/dubbo-serialization-gson/pom.xml
+++ b/dubbo-serialization/dubbo-serialization-gson/pom.xml
@@ -19,7 +19,7 @@ Licensed to the Apache Software Foundation (ASF) under one or more
     <parent>
         <artifactId>dubbo-serialization</artifactId>
         <groupId>org.apache.dubbo</groupId>
-        <version>2.7.2-SNAPSHOT</version>
+        <version>${revision}</version>
     </parent>
     <artifactId>dubbo-serialization-gson</artifactId>
     <packaging>jar</packaging>

